### DISCUSSION TITLE
Исчезает селект при вызове linktoinputselect из временной таблицы

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#545] [ОШБК!] [answered]
Странное поведение системы при вызове linktoinputselect из временной таблицы. Либо исчезает селект, либо не закрывается всплывающее окно после выбора селекта.

https://youtu.be/lzs6NCC_R9I

**Comment by ttm-alexeypolunin** [comment]
Похоже исправление этой фигни не попало в текущую V4

**Reply by mihail-cornev** [reply]
Исправлений и обновлений V4 больше не будет?

**Reply by ttm-alexeypolunin** [reply]
Да, не будет

**Reply by ttm-alexeypolunin** [reply, marked as answer]
Мне кажется исправили в V5
